rs/nervous_system/integration_tests/tests/constraints_dependencies.rs

```diff
@@ -11,11 +11,14 @@ use ic_sns_init::{
 // against the sum of intermediate limits set for various types of SNS neurons. These intermediate
 // limits are not checked within just one canister, so testing their inter-consistency is done here.
 //
+// Many SNS neurons may be created after a swap succeeds. The number of such neurons is limited to
+// `MAX_NEURONS_FOR_DIRECT_PARTICIPANTS`. This limit is enforced only *during* the swap. In effect,
+// this limits the maximum number of swap participants to `MAX_NEURONS_FOR_DIRECT_PARTICIPANTS` /
+// #number of SNS neurons per participant (a.k.a., the SNS basket count).
```

Comment:
NCR, just wanted to say I was confused by this comment here because the test below doesn't seem to be about this.
The test below only checks `MAX_NEURONS_FOR_DIRECT_PARTICIPANTS` and it doesn't seem relevant here how that's enforced as long as it is.

Reply:
I think I agree, but more context has been requested in one of the previous code reviews. 

So I'm going to go with the NC option.